doc/site/en/guides/border-router/beaglebone-black.md

```diff
@@ -1,338 +1,204 @@
-
-1.  Delete the existing Linux partition without deleting the data. You have
-    removed the picture frame, but the picture is still present on the canvas.
-1.  Create a new Linux partition that starts exactly where the old one started,
-    but ends close to the end of the SD card. You have created a larger picture
-    frame. The picture—the data—is still there on the canvas. It has not moved
-    and was not corrupted by this operation.
-1.  Use a file system-specific tool to grow the file system within the bounds of
-    the new partition. The canvas is stretched to fill the new, larger picture
-    frame.
-
-### 1. Identify the current data partition
-
-Boot the BBB from the SD Card and log in as `root`:
+*  External 5V AC adapter for power
+*  An 8 GB or larger microSD card ("uSD card" in this guide)
+*  A supported OpenThread platform (such as the [TI
+   CC2652](https://openthread.io/vendors/texas-instruments#cc2652)) for Thread
+   network connectivity in an RCP design
 
-```
-$ sudo bash
-```
+Steps to enable
+1. Download and install the OS.
+1. Prepare the Debian Environment for OTBR
+1. Build and install OTBR
+1. Set up a Wi-Fi access point
 
-Identify the SD card data partition. The `p1` suffix on the `Filesystem`
-field is the naming convention for Partition 1. The device itself is
-`/dev/mmcblk0`. In this example, only 295 MB are free. This is not enough
-space to build and install OTBR.
+Note: The BBB does not have built-in Wi-Fi support. This guide was built and
+tested with a BBONE-GATEWAY-CAPE for Wi-Fi AP operation. Some BeagleBone
+variants have onboard Wi-Fi capability, and some of this guide may be
+applicable.
 
-```
-root@beaglebone:/home/debian# df -hT /
-Filesystem     Type  Size  Used Avail Use% Mounted on
-/dev/mmcblk0p1 ext4  3.3G  2.8G  295M  91% /
-```
+## Step 1: Download and install the OS.
 
-> Note: This example image has a single partition,
-  other images may have additional partitions.
+1. Download the latest Debian IoT image for BeagleBone. [beaglebone latest
+   images](https://beagleboard.org/latest-images)
+   *  The version used for this guide was
+      `bone-debian-10.3-iot-armhf-2020-04-06-4gb.img.xz`
+1. Install the OS image on a uSD Card by following the [BeagleBone getting
+   started guide](https://beagleboard.org/getting-started).
+1. Boot the BeagleBone and SSH into the device.
+   *  Connectivity over a local Ethernet based network is recommended.
+   *  The cloud9 IDE will be disabled later in this guide.
+   *  This guide will change the state of BeagleBone network interfaces, be
+      aware you secure shell may disconnect.
+   *  Modern BeagleBone bootloaders will run from the uSD card by default, but
+      some BeagleBone Black devices may try to boot from the internal eMMC.
+      Make sure to press the BOOT Button in this case.
 
-### 2. Create the new, larger partition
+Warning: The power requirements of the development kit used for the OpenThread
```

Comment:
Callouts should be blockquote with > preceding the first line and the others indented

> Warning:

Reply:
didn't look at the raw version of the style guide, I lost the block quotes on the github rendered version.